### DISCUSSION TITLE
Akira installation process

### DISCUSSION THREAD

**Discussion by LucileDT** [akiraux/Akira#572] [Q&A]
I saw [here](https://github.com/akiraux/Akira/issues/460) that Akira seems to work differently if compiled or installed from snap/flatpack.

I was wondering if we have some documentation about the different versions of Akira which are currently available (ordered by installation process)? I think it would be a great tool both for users (to know how they can get Akira) and for devs (to know what installer they need to update when releasing a new version) :slightly_smiling_face:

**Comment by Alecaddd** [comment]
Indeed, our installation documentation is not the best, but that's mostly because the app is in the early stages and not properly released on easily accessible channels.
It can be installed via Snap and Flathub, but both of those are behind the edge/beta channel, so not super publicly available.
The issue about the language is being tracked by @bilelmoussaoui, which is our FlatPak expert.

I think I will do a proper rework of all the documentation once we're close to the first beta release as for that time we will have properly fledge out all the dependencies and quirks of running the app.

**Reply by LucileDT** [reply]
Do you have a list of what to put in the reworked documentation?
